src/modules/bookings/components/booking-card.tsx

```diff
@@ -0,0 +1,95 @@
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+import Chip from '@mui/material/Chip';
+import Divider from '@mui/material/Divider';
+import CalendarIcon from '@mui/icons-material/CalendarMonth';
+import PersonIcon from '@mui/icons-material/Person';
+import LocationIcon from '@mui/icons-material/LocationOn';
+import PriceIcon from '@mui/icons-material/AttachMoney';
+
+import { type Booking } from '../types/booking.type';
+import { type Accommodation } from '../../home/types/accommodation.type';
+import { bookingCardStyles } from './booking-card.styles';
+import { useNavigate } from 'react-router-dom';
+import { BookingStatus } from '../types/booking-status.constant';
+
+type BookingsWithAccommodation = Booking & {
+  accommodation: Accommodation;
+};
+
+type BookingCardProps = {
+  booking: BookingsWithAccommodation;
+};
+
+export function BookingCard({ booking }: BookingCardProps) {
+  const navigate = useNavigate();
+
+  const formatDate = (date: Date) => {
+    return new Date(date).toLocaleDateString('uz-UZ', {
+      year: 'numeric',
+      month: 'long',
+      day: 'numeric',
+    });
+  };
+
+  const getStatusColor = (status: BookingStatus) => {
```

Comment:
you can use switch case intead of else if

Reply:
> you can use switch case intead of else if

Or ternary operator